src/tests/multihost/alltests/tevent_loop.c

```diff
@@ -0,0 +1,52 @@
+#include <tevent.h>
+#include <talloc.h>
+#include <stdio.h>
```

Comment:
This file should be in alltest/src directory

Reply:
Is it alltests/src or alltests/script?

Reply:
https://github.com/SSSD/sssd/tree/master/src/tests/multihost/alltests/script

Reply:
moved there